google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -196,9 +208,19 @@ public Table createTable(CreateTableRequest request) {
    * <p>Sample code:
    *
    * <pre>{@code
+   * // A table with a single column family, which retains values up to 7 days.
    * ApiFuture<Table> tableFuture = client.createTableAsync(
    *   CreateTableRequest.of("my-table")
-   *     .addFamily("cf", GCRules.GCRULES.maxVersions(1))
+   *     .addFamily("cf", GCRULES.maxAge(Duration.ofDays(7)))
+   * );
+   *
+   * // Another table with more complex garbage collection rules.
+   * ApiFuture<Table> tableFuture = client.createTableAsync(
+   *   CreateTableRequest.of("my-table")
+   *     .addFamily("cf", GCRULES.intersection()
+   *       .rule(GCRULES.maxAge(120, TimeUnit.HOURS))
+   *       .rule(GCRULES.maxVersions(10))
+   *     )
```

Comment:
extra paren

Reply:
I followed the similar pattern present in the file, _To close paran inline if arguments are inlined. In the case of multiline arguments, close the operation in the new line._
Please let me know if I should reformat this sample code.

Reply:
nope, my mistake